app/client/src/ee/components/formControls/Rag/RagIntegrations.tsx

```
@@ -0,0 +1,7 @@
+interface RagIntegrationsProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)